src/engine/Test_engine.ml

```diff
@@ -83,7 +83,8 @@ let xtarget_of_file (xlang : Xlang.t) (target : Fpath.t) : Xtarget.t =
           assert false)
   in
   {
-    Xtarget.file = target;
+    Xtarget.source = File target;
```

Comment:
You don't need the `Xtarget.` field prefix here. The compiler knows because the return type of the function is set with the annotation `let ... : Xtarget.t = ...`.

Reply:
Sure; I can remove this.

Reply:
Done in 135dff8da0424f9a79b39d7a72e2c7c10936e403